ui/src/assets/i18n/de.json

```diff
@@ -0,0 +1,1082 @@
+      "cloneTitle2": "Klone Aufgaben",
+      "content": "Diese Aktion wird die folgende <strong>Aufgabe(n)</strong> erstellen",
+      "duplicate": "Doppelter Aufgaben-Name, bitte überprüfen Sie die Eingabe-Namen.",
+      "requiredName": "Aufgaben-Name wird benötigt.",
+      "invalidName": "Ungültiger Aufgaben-Name (keine Leer- oder Sonderzeichen).",
+      "maxLengthName": "Aufgaben-Name muss weniger als 255 Zeichen lang sein.",
+      "uniqueName": "Aufgaben-Name wird bereits verwendet.",
+      "cloning": "Klone gerade Aufgabe(n)",
+      "clone": "Klone die Aufgabe",
+      "clone2": "Klone die Aufgaben",
+      "message": {
+        "partialSuccessTitle": "Aufgabe(n) klonen",
+        "partialSuccessContent": "Aufgabe(n) wurden teilweise geklont",
+        "successTitle": "Aufgabe(n) klonen",
+        "successContent": "Aufgabe(n) wurden erfolgreich geklont",
+        "errorContent": "Keine Aufgabe(n) geklont."
+      }
+    },
+    "destroy": {
+      "destroyTitle": "Bestätige das Zerstören der Aufgabe",
+      "destroyTitle2": "Bestätige das Zerstören der Aufgaben",
+      "content": "Diese Aktion zerstört und löscht die Aufgabe <strong>{{ name }}</strong>. Sind Sie sicher?",
+      "content2": "Diese Aktion wird die unterhalb aufgeführten<strong>{{ count }} Aufgaben</strong> zerstören. Sind Sie sicher?",
+      "destroying": "Zerstöre Aufgabe(n)",
+      "destroy": "Zerstöre die Aufgabe",
+      "destroy2": "Zerstöre die Aufgaben",
+      "message": {
+        "successTitle": "Zerstöre die Aufgabe",
+        "successContent": "Die Aufgabe \"{{ name }}\" wurde erfolgreich entfernt.",
+        "successTitle2": "Zerstöre die Aufgaben",
+        "successContent2": "{{ count }} Aufgaben-Definitionen zerstört.",
+        "errorContent": "Ein Fehler ist beim Löschen von mehreren Aufgaben aufgetreten. Bitte prüfen Sie die Server-Logs für mehr Informationen."
+      }
+    },
+    "task": {
+      "title": "Aufgabe",
+      "launch": "Starte Aufgabe",
+      "schedule": "Aufgaben einplanen",
+      "destroy": "Zerstöre Aufgabe",
+      "information": "Informationen",
+      "applications": "Anwendungen",
+      "showPipeline": "Zeige Daten-Pipeline",
+      "lastExecution": "Letzte Ausführung",
+      "viewTaskExecution": "Zeige Aufgaben-Ausführung",
+      "viewLog": "Zeige Log",
+      "latestExecutions": "Neuesten Ausführung(en)",
+      "schedules": "Zeitpläne"
+    },
+    "create": {
+      "title": "Erstelle eine Aufgabe",
+      "createTask": "Erstelle Aufgabe",
+      "modal": {
+        "creating": "Erstelle gerade eine Aufgabe",
+        "create": "Erstelle eine Aufgabe",
+        "taskName": "Aufgaben-Name",
+        "taskDescription": "Aufgaben-Beschreibung",
+        "content": "Diese Aktion wird die folgende <strong>Aufgabe</strong> erstellen:",
+        "duplicateName": "Doppelter Aufgaben-Name, bitte prüfen Sie die Eingabe-Namen.",
+        "maxLengthName": "Der Aufgaben-Name muss weniger als 256 Zeichen lang sein.",
```

Comment:
Really 256 and not 255 ?

Reply:
Good question sometimes it is 255 and sometimes it is 256. I switch all occurrences to 255.